Provisioner/BuildProvisioner.php

```diff
@@ -204,10 +198,10 @@ protected function createFiles(SSH2 $ssh, ApplicationEnvironment $applicationEnv
 
                 $this->executeSshCommand($ssh, $command);
 
-                $this->taskService->addSuccessLogMessage($this->task, 'Files created.', 2);
+                $this->taskLoggerService->addSuccessLogMessage($this->task, 'Files created.', 2);
             }
         } catch (\Exception $ex) {
-            $this->taskService
+            $this->taskLoggerService
                 ->addErrorLogMessage($this->task, $ex->getMessage(), 2)
                 ->addFailedLogMessage($this->task, 'Creating files failed.', 2);
 
```

Comment:
throw logged exception

Reply:
This exception is caught in the `doRun` method and re-thrown as a LoggedException there.